addons/medical/functions/fnc_setUnconscious.sqf

```diff
@@ -46,6 +46,16 @@ if (currentWeapon _unit != primaryWeapon _unit) then {
     _unit selectWeapon primaryWeapon _unit;
 };
 
+if (GVAR(dropWeaponOnUnconscious) != 0) then {
+    if (_unit == ACE_player) then {
+        if (_knockOut) then {
+            if (random 1 <= GVAR(dropWeaponOnUnconscious)) then {
+                ACE_player call EFUNC(hitreactions,throwWeapon);
+            };
+        };
+    };
+};
```

Comment:
Why not to use &&? And there is already `if (_knockOut)` couple lines lower. So why not to put it inside?
```sqf
if (GVAR(dropWeaponOnUnconscious) != 0 && {_unit == ACE_player && {random 1 <= GVAR(dropWeaponOnUnconscious)}}) then {
    ACE_player call EFUNC(hitreactions,throwWeapon);
};
```

Reply:
agree way better will adjust.